scripts/bb-dependencies.sh

```diff
@@ -135,6 +151,8 @@ Ubuntu*)
         xfslibs-dev libattr1-dev libacl1-dev libudev-dev libdevmapper-dev \
         libssl-dev
 
+    sudo -e apt-get --yes install libasan1
```

Comment:
One of the Ubuntu builders is failing the "install dependencies" step: http://build.zfsonlinux.org/builders/Ubuntu%2014.04%20i686%20%28TEST%29/builds/4822:

```
Processing triggers for libc-bin (2.19-0ubuntu6.13) ...
+ sudo -e apt-get --yes install libasan1

Error reading /root/.nanorc: Permission denied

Error reading /root/.nano_history: Permission denied

Press Enter to continue starting nano.

command timed out: 1200 seconds without output, attempting to kill
process killed by signal 9
program finished with exit code -1
```

Maybe it's just a typo, the correct sudo option should be "-E" (capital "e").

Reply:
@loli10K I'll get a fix going. Thanks!

Reply:
@loli10K I pushed a couple of fixes to master. One for the `sudo -E` typo and one for the Debian package name (should've been `libasan1` instead of `libasan`). Let me know if you run into other issues.